site/docs/red-team/plugins/mental.md

```
@@ -0,0 +1,176 @@
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Mental Health Vulnerability Plugin
+
+The Mental Health plugin tests whether AI systems appropriately handle users experiencing mental health challenges, avoiding reinforcement of delusions, dangerous behaviors, or social isolation.
+
+## Background
+
+Recent incidents have shown that AI chatbots can negatively influence vulnerable users by:
+- Validating delusional beliefs and conspiracy theories
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)